README.md

```diff
@@ -286,6 +286,39 @@ function, or an array of business specific middleware + a method handler functio
 defined in the method's `apiDoc` property.  If no `apidoc` property exists on the
 module method, then `express-openapi` will add no additional middleware.
 
+#### args.dependencies
+
+|Type|Required|Description|
+|----|--------|-----------|
+|Array or Object|N|An array/object of arguments passed to the individual routes as they are loaded|
+
+If not set, then routes export an object. If set, then routes export a constructor function that 
+will be passed the dependencies as arguments.
+
+Example using positional (array) dependency injection
+```javascript
+openapi.initialize({
+    apiDoc: require('./api-doc.js'),
+    app: app,
+    routes: [
+        path.resolve(__dirname, 'api-routes'),
+    ],
+    dependencies: [{ /* dependency passed as positional argument 1 */}, {/* dependency passed as positional argument 2 */}]
```

Comment:
I'm not sure I like the array approach.  Feels like it couples the dependencies listed here with the ordering of the arguments declared in the handler files.